pkgs/by-name/bs/bsync/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation {
+  pname = "bsync";
+  version = "20231221-25f7773";
+
+  src = fetchFromGitHub {
+    owner  = "dooblem";
+    repo   = "bsync";
+    rev    = "25f77730750720ad68b0ab2773e79d9ca98c3647";
+    hash   = "sha256-k25MjLis0/dp1TTS4aFeJZq/c0T01LmNcWtC+dw/kKY=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm555 bsync -t $out/bin
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/dooblem/bsync";
+    description = "Bidirectional Synchronization using Rsync";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers;  [ "dietmarw" ];
```

Assuming you've rebased this branch on top of a version of master that contains the commit where you add yourself as a maintainer, this line should look like this:

```suggestion
    maintainers = with maintainers;  [ dietmarw ];
```

Feel free to ask if you're not used with this kind of git usage.